app/src/main/java/ru/chaichuk/hwapp/ActorsListAdapter.kt

```diff
@@ -0,0 +1,79 @@
+package ru.chaichuk.hwapp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.content.res.ResourcesCompat
+import androidx.recyclerview.widget.RecyclerView
+import ru.chaichuk.hwapp.data.models.Actor
+
+class ActorsListAdapter() : RecyclerView.Adapter<ActorsViewHolder>() {
+    private var actors = listOf<Actor>()
+
+    override fun getItemViewType(position: Int): Int {
+        return when (actors.size) {
+            0 -> VIEW_TYPE_EMPTY
```

Comment:
why need VIEW_TYPE_EMPTY view type ?

Reply:
copied it from workshop code to understand why I could need it in my code.
did not find any reason, but leaved it to think about it later

Reply:
let's remove unused codes from PR
it's not good habbit

Reply:
removed all that empty view holders